librz/analysis/p/analysis_x86_cs.c

```diff
@@ -742,6 +742,13 @@ static void anop_esil(RzAnalysis *a, RzAnalysisOp *op, ut64 addr, const ut8 *buf
 				dst, width, src, width, dst);
 		}
 		break;
+	// comiss
+	case X86_INS_COMISS:
+	case X86_INS_UCOMISS:
+	case X86_INS_VCOMISS:
+	case X86_INS_VUCOMISS:
+		op->type = RZ_ANALYSIS_OP_TYPE_COMISS;
```

Comment:
this should probably be `RZ_ANALYSIS_OP_TYPE_SIMD | RZ_ANALYSIS_OP_TYPE_CMP`

Reply:
You're right, since ucomiss instruction is used to compare floating point values, we should give it 
RZ_ANALYSIS_OP_TYPE_SIMD = 0x04000000.  And this RZ_ANALYSIS_OP_TYPE_SIMD | RZ_ANALYSIS_OP_TYPE_CMP too. Let me change them

Reply:
Done